### ISSUE TITLE
issue with ingressclasses and .Values.scope.singleNamespace

### ISSUE THREAD

**Issue by gabrielsyapse** [datawire/ambassador-chart#161]
I am using `.Values.scope.singleNamespace` so this is a role, not a clulsterrole.
https://github.com/datawire/ambassador-chart/blob/master/templates/rbac.yaml#L3-L7

However, this fix is for a clusterrole:
https://github.com/datawire/ambassador/issues/3136

But this apiresource is clusterwide, not per namespace, so this doesn't work for me with a role.
```
- apiGroups: [ "extensions", "networking.k8s.io" ]
  resources: [ "ingressclasses" ]
  verbs: ["list"]
```

**Comment by gabrielsyapse** [comment]
When I reported this, I was trying chart version "6.5.16" which fails to deploy with errors about cluster scope. However, version "6.5.18" does work.

**Comment by chris-ng-scmp** [comment]
yes, have the same issue for singleNamespace installation since `tag: 1.10.0`

The RBAC created for singleNamespace is Role, not ClusterRole, in singleNamespace I think should not require such permission in cluster level

**Comment by chris-ng-scmp** [comment]
> yes, have the same issue for singleNamespace installation since `tag: 1.10.0`
> 
> The RBAC created for singleNamespace is Role, not ClusterRole, in singleNamespace I think should not require such permission in cluster level

Ah, actually the issue is fixed in 1.11.0
